apps/ui/src/networks/common/graphqlApi/index.ts

```diff
@@ -84,6 +84,7 @@ function formatSpace(space: ApiSpace, networkId: NetworkID): Space {
   return {
     ...space,
     network: networkId,
+    chain_id: CHAIN_IDS[networkId],
```

Comment:
This is not going to map to anything for Starknet spaces. I'd rather call it `snapshot_chain_id` and make it nullable and do that for networks that use its own snapshot chain.